app/__init__.py

```diff
@@ -87,18 +90,24 @@ def after_request(response):
     return response
 
 
-def _get_suggestions(data_set_source, query):
+def _get_suggestions(data_set_source, query, strategy):
     """Get suggestions.
 
     Get suggestions for a given query string.
 
     :param (str) data_set_source: Data set to query.
     :param (str) query: Query string.
+    :param (str) strategy: Search strategy to use.
     :returns (list): List of 10 candidates
     """
-    g = guess.Guess(data_set_source)
+    if strategy not in ALL_STRATEGIES:
+        abort(400)
+    if strategy == simple.STRATEGY_NAME:
```

Comment:
@davecharles Shouldn't this be an `elif` statement?

Reply:
No I intended that, however now you mention it, this might be better (do away with `ALL_STRATEGIES`):
```python
if strategy == simple.STRATEGY:
    ...
elif strategy == guess.STRATEGY:
    ...
else:
    abort(400)
```